src/rust/runtime/mod.rs

```diff
@@ -99,6 +99,12 @@ use self::{
     },
 };
 
+//======================================================================================================================
+// Constants
+//======================================================================================================================
+
+const TIMER_RESOLUTION: usize = 64;
```

Comment:
Could you add a comment here that explains the rationale for setting `TIMER_RESOLUTION` to 64?